Game/Game/Program.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using Game.classes;
+using Game.enums;
+using Game.interfaces;
+
+namespace Game
+{
+    public class Program
+    {
+        private const int MIN_PLAYERS = 2;
+        private const int MAX_PLAYERS = 8;
+        private const int MAX_ATTEMPTS = 100;
+        
+        public static void Main(string[] args)
+        {
+            InitGame();
+        }
+
+        private static void InitGame()
+        {
```

Comment:
move it to the separate class
It isn't a responsibility of this class
SOLID principles
https://www.codeproject.com/Tips/1033646/SOLID-Principle-with-Csharp-Example

Reply:
Fixed